### PR TITLE
changes in merchant orders

### PR DESCRIPTION
**Pull request by fbielli**
Added get search endpoint and curl and json responses to all endpoints.

### CODE REVIEW

File: reference/merchant_orders/resource.yaml [hunk 1]
```diff
@@ -375,9 +380,14 @@ properties:
...
         description:
-          en: Total amount paid by the buyer.
-          es: Monto total pagado por el comprador.
-          pt: Montante total pago pelo comprador.
+          en: Item quantity.
+          es: Cantidad de items.
+          pt: Item quantity.
+        enum:
+          - put:
+              en: This is the only field of item that can be modiffied in put
```
Comment (adfonseca) [inline reference/merchant_orders/resource.yaml]: The correct isn't **modified**?

Comment (fbielli) [reply reference/merchant_orders/resource.yaml]: Si, está solucionado. Si te fijás te marca el archivo como "Outdated".

File: reference/merchant_orders/samples/get_curl_response.json [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  "status": "closed",
+  "external_reference": "default",
+  "preference_id": "Preference identification",
+  "payments": [
+    {
+      "id": 9999999999,
+      "transaction_amount": 1,
+      "total_paid_amount": 1,
+      "shipping_cost": 0,
+      "currency_id": "ARS",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/get_curl_response.json]:
```suggestion
      "currency_id": "[FAKER][CURRENCY][ACRONYM]",
```

File: reference/merchant_orders/samples/get_curl_response.json [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        "zip_code": "9999",
+        "street_name": "Street name",
+        "street_number": "999",
+        "floor": null,
+        "apartment": null
+      },
+      "shipping_option": {
+        "id": 999999999,
+        "cost": 0,
+        "currency_id": "ARS",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/get_curl_response.json]:
```suggestion
        "currency_id": "[FAKER][CURRENCY][ACRONYM]",
```

File: reference/merchant_orders/samples/get_curl_response.json [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  "refunded_amount": 0,
+  "payer": {
+    "id": 999999999,
+    "email": "payer@email.com"
+  },
+  "items": [
+    {
+      "id": "item id",
+      "category_id": "item category",
+      "currency_id": "ARS",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/get_curl_response.json]:
```suggestion
      "currency_id": "[FAKER][CURRENCY][ACRONYM]",
```

File: reference/merchant_orders/samples/get_search_curl_response.json [hunk 5]
```diff
@@ -1,30 +1,97 @@
...
+            "zip_code": "9999",
+            "street_name": "Street name",
+            "street_number": "999",
+            "floor": null,
+            "apartment": null
+          },
+          "shipping_option": {
+            "id": 999999999,
+            "cost": 0,
+            "currency_id": "ARS",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/get_search_curl_response.json]:
```suggestion
            "currency_id": "[FAKER][CURRENCY][ACRONYM]",
```

File: reference/merchant_orders/samples/post_curl.sh [hunk 6]
```diff
@@ -0,0 +1,20 @@
+curl -X POST \
+  'https://api.mercadopago.com/merchant_orders?access_token=ACCESS_TOKEN_ENV' \
+  -d '{
+    "preference_id": "preference_id",
+    "items": [
+        {
+            "id": "item_id",
+            "category_id": "category_id",
+            "currency_id": "ARS",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/post_curl.sh]:
```suggestion
            "currency_id": "[FAKER][CURRENCY][ACRONYM]",
```

File: reference/merchant_orders/samples/post_curl_response.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  "refunded_amount": 0,
+  "payer": {
+    "id": 999999999,
+    "email": "payer@email.com"
+  },
+  "items": [
+    {
+      "id": "item id",
+      "category_id": "item category",
+      "currency_id": "ARS",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/post_curl_response.json]:
```suggestion
      "currency_id": "[FAKER][CURRENCY][ACRONYM]",
```

File: reference/merchant_orders/samples/get_curl_response.json [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+    "nickname": null
+  },
+  "marketplace": "NONE",
+  "notification_url": null,
+  "date_created": "2018-09-14T13:11:31.000-04:00",
+  "last_updated": "2018-09-14T13:11:43.000-04:00",
+  "sponsor_id": null,
+  "shipping_cost": 0,
+  "total_amount": 5,
+  "site_id": "MLA",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/get_curl_response.json]:
```suggestion
  "site_id": "[FAKER][GLOBALIZE][UPPER_SITE_ID]",
```

File: reference/merchant_orders/samples/post_curl_response.json [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+    "nickname": null
+  },
+  "marketplace": "NONE",
+  "notification_url": null,
+  "date_created": "2018-09-14T13:11:31.000-04:00",
+  "last_updated": "2018-09-14T13:11:43.000-04:00",
+  "sponsor_id": null,
+  "shipping_cost": 0,
+  "total_amount": 5,
+  "site_id": "MLA",
```
Comment (Juanii95) [inline reference/merchant_orders/samples/post_curl_response.json]:
```suggestion
  "site_id": "[FAKER][GLOBALIZE][UPPER_SITE_ID]",
```

**Comment by adfonseca** [comment]
Thanks for your contribution @fbielli! our team will review it and notify you through this channel

**Pull request merged**